### ISSUE TITLE
Stripped binaries in dataset cause bad ground truth comparisons

### ISSUE THREAD

**Issue by aeflores** [junxzm1990/x86-sok#32]
I have noticed that several binaries in the dataset are stripped:
e.g.
```
mthumb_executables/clients/gcc_O2/scp
mthumb_executables/clients/gcc_O2/sftp
mthumb_executables/clients/gcc_O2/ssh
mthumb_executables/clients/gcc_O2/ssh-add
...
```
since the comparison scripts use symbols to exclude functions added by linker (https://github.com/junxzm1990/x86-sok/blob/master/compare/compareInstsArmMips.py#L504), this can cause problem in these binaries.

For example, in `mthumb_executables/clients/gcc_O2/scp`, the address `0x28c4` does not seem to belong to the ground truth, however, that is the entrypoint address (the entrypoint is `0x28c5` according to readelf since this is a thumb binary).
In other words, the `_start` function is missing from the ground truth and it cannot be excluded because the binary is stripped.
Do you have any advice on how to deal with these cases?

**Comment by bin2415** [comment]
We have recompiled the binaries related to `ssh` in Thumb mode. 

The binaries are shared in [google drive](https://drive.google.com/file/d/1c8E3UWRd419U2vGI12G7JJy8DvrEvNc-/view?usp=sharing)
